code/modules/projectiles/radbolt.dm

```diff
@@ -34,5 +34,7 @@ toxic - poisons
 	hit_ground_chance = 50
 	//Can we pass windows
 	window_pass = 1
-	//no visible message upon bullet_act
+	//no visible message upon bullet_act and no armor block message
 	silentshot = 1
+	//Bullet equivalent impact_range
+	impact_range = 0
```

Comment:
no need to move this here

Reply:
Ok. I understood it was not good to have the datum information in two places.